src/main/java/tech/sourced/siva/RangeInputStream.java

```diff
@@ -0,0 +1,56 @@
+package tech.sourced.siva;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+/**
+ *
+ */
```

Comment:
Should there be something in these comments? Otherwise they might as well be removed.